tox.ini

```diff
@@ -1,19 +1,30 @@
 [tox]
 envlist = py{36,37,38},pypy3,codechecks
 
+[docker:quay.io/redhatqe/selenium-standalone]
+image =
+    quay.io/redhatqe/selenium-standalone:latest
+ports =
+    4444:4444/tcp
+    4444:4444/udp
+volumes =
+    /dev/shm:/dev/shm
+
 [testenv]
 passenv = BROWSER
-deps=
-    pytest==4.6.4
+deps =
+    pytest==5.4.3
     pytest-localserver==0.5.0
-    pytest-cov==2.7.1
+    pytest-cov==2.10.1
     anytree
+docker =
+   quay.io/redhatqe/selenium-standalone
 commands =
-    py.test {posargs: -v --cov widgetastic --cov-report term-missing --cov-report xml --no-cov-on-fail -s}
+    py.test {posargs: -v -x --cov widgetastic --cov-report term-missing --cov-report xml --no-cov-on-fail -s}
```

Comment:
I don't plan on keeping `-x` arg, but was using it locally because I wanted to stop after the first failure.